lib/Net/Braintree/TransactionGateway.pm

```diff
@@ -42,6 +42,7 @@ sub void {
 
 sub refund {
   my ($self, $id, $params) = @_;
+  confess 'ArgumentError' unless verify_params($params, refund_signature);
```

Comment:
We use double quotes in many other places—could we change this to be double quotes?